### DISCUSSION TITLE
File does not exist : configs/yolov3-kFLOAT-batch1.engine

### DISCUSSION THREAD

**Discussion by blackHorz** [enazoe/yolo-tensorrt#91] [General]
I am trying to run it on Jetson Nano with following instructions

git clone https://github.com/enazoe/yolo-tensorrt.git
cd yolo-tensorrt/
mkdir build
cd build/
cmake ..
make
./yolo-trt

First I chose the CMakeLists.txt as mentioned in the hyperlink before this text. However in this case it could not found the class_timer.hpp. Later I used the CMakeLists.txt file from the repository. 

I ran 
cmake . && make
./yolo-trt

To which it is giving following error:
File does not exist : configs/yolov3-kFLOAT-batch1.engine

**during make process I had few deprecated warnings.

**Comment by enazoe** [comment]
is mormal
